targets_test.go

```diff
@@ -39,7 +39,7 @@ func TestOKKubeConform(t *testing.T) {
 
 // Test for manifest files expected to fail Kyverno policy validation
 func TestFailedValidateKyverno(t *testing.T) {
-	path := "tests/templates/validate-fail/"
+	path := "tests/templates/validate-fail/deployment-fail.yaml"
```

Comment:
maybe you can add an other file here to test the splitting and creating the 'resourceArgs'